interview-questions.md

```diff
@@ -8,51 +8,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What are props in React?
 
   Your answer:
+  --> Props is the way data is sent in React.
 
   Researched answer:
+  -->Props(short for properties) are used to pass read-only data which means it cannot be modified.
 
 
 
 2. What is a DOM event?
 
   Your answer:
+  -->A DOM event is an event mapped to a browser that effects what happens in the browser.
 
   Researched answer:
+  --> DOM events are triggered by user interactions signaling that something is occuring.
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
   Your answer:
+  -->Object-oriented programing is storaged based programing which means info is stored in the object.
 
   Researched answer:
+  -->Object oriented programing is based on objects and classes, Functional programming relies on functions.
 
 
 
 4. What is the difference between a Float and an Integer in Ruby?
 
   Your answer:
+  -->Both are numbers but an INTEGAR is a whole number and a FLOAT is a number with a decimal.  
+  Example: 
+  4 is an Integar
+  4.1 is a FLOAT
 
   Researched answer:
+  --->Floats andn integers are both types of numerical data but only floats use decimal points and integars are whole numbers.
 
 
 
 5. Ruby has an implicit return. What does that mean?
 
   Your answer:
+  -->Implicit return would mean you wouldn't need to specify a return at the end of your code since you already stated the desired outcome somewhere else in the code.
 
   Researched answer:
+  -->Implicit return is when a function is returned without using a return keyword.
```

Comment:
While the answers in this section are accurate, there is a lot of room to expand further on all of your answers.  This currently is meeting the bare minimum requirements for this section.  Answers in this section should be thorough and mimic how you would answer in an interview.  These answers currently may come off to an interviewer that you are uninterested.